docs/rules/coding-convention.md

```
@@ -0,0 +1,45 @@
+# Server Coding Convention
+
+  ## Package Naming
```

markdownlint-cli2 (0.22.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)